docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
@himanshug What I was thinking is that it seems like we have two pretty similar operations,

- extracting nested data from plain Java Maps/Lists- which is what json-path does, we're actually not using it on the JSON directly
- extracting nested data from avro records

It would be nice if the way you express these extractions is similar. That way knowledge of one would transfer to the other and people that use both json and avro wouldn't need to learn both. The implementations don't need to be shared, although that would be cool too if possible.

I am not really familiar with avro so I might be missing reasons why this doesn't make sense or isn't worth it.